redfish-core/lib/power_supply.hpp

```diff
@@ -622,6 +492,14 @@ inline void requestRoutesPowerSupply(App& app)
                                     "/redfish/v1/Chassis/" + chassisID +
                                     "/PowerSubsystem/PowerSupplies/" +
                                     powerSupplyID;
+                                if (chassisID == "chassis")
```

Comment:
You are hardcoding the chassis id and checking it to determine if the PowerSupplyMetrics should be there.. It would take too long to add an association from the metrics to the powersuppy ?